examples/AdvancedOperations/AddGmailAd.php

```diff
@@ -121,11 +121,11 @@ public static function runExample(
     }
 
     /**
-     * Adds the image files.
+     * Adds the media files based on URLs.
```

Comment:
Nit: "... media files by using the class constants."

Reply:
Done.